web/annotation_editor_layer_builder.css

```
@@ -266,7 +266,7 @@
       & > .resizer {
         width: var(--resizer-size);
         height: var(--resizer-size);
-        background: var(--resizer-bg-color);
+        background-color: var(--resizer-bg-color);
         background-clip: content-box;
```

Could we make this a one-liner instead, or would that affect some properties that we don't want?
```suggestion
        background: content-box var(--resizer-bg-color);
```